lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,559 @@
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Can only provide one replacing extension</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: Performing Location Extension
+    required: 'No'
+    type: Extension
+    description: A location where the charge was performed.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "resource": {
+              "extension": [
+                {
+                  "valueReference": {
+                    "reference": "Location/2170691494",
+                    "display": "RCR Analytics"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `valueReference`</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: Description Extension
+    required: 'No'
+    type: Extension
+    description: A description providing additional details of the resource.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "resource": {
+              "extension": [
+                {
+                  "valueString": "Acetaminophen Level",
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/description"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `valueString`</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: Custom Attribute Extension
+    required: 'No'
+    type: List of Custom Attribute Extension
+    description: A client defined custom attribute for the charge.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          {
+            "resource": {
+              "extension": [
+                {
+                  "extension": [
+                    {
+                      "id": "17433932",
+                      "valueString": "Line Note Text",
+                      "url": "custom-attribute-name"
+                    },
+                    {
+                      "valueString": "Raghu, P",
+                      "url": "custom-attribute-value"
+                    }
+                  ],
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Custom-attribute-value and Custom-attribute-value are both required</li>
+        <li>Custom-attribute-value can be of type integer, string, decimal, or date</li>
+      </ul>
+    url: http://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
```

Comment:
I think we want to use `https` wherever possible

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/c8a1c8272a99dbf6bd094d88ad599ff945126235#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R389